UI/EmuScreen.cpp

```diff
@@ -70,10 +70,22 @@
 #include "Windows/MainWindow.h"
 #endif
 
+bool frameStep_;
```

Comment:
Okay, but in that case they should be `static`.  That means if some other `.cpp` file has a `lastNumFlips` (like say, some cheating or networking feature, or some other TAS code), it won't be the same variable.

-[Unknown]